datastore/elastic_search/query.py

```diff
@@ -176,22 +180,29 @@ def _generate_es_ngram_search_dictionary(entity_name, ngrams_list, fuzziness_thr
         entity_name: name of the entity to perform a 'term' query on
         ngrams_list: list of ngrams to perform fuzzy search for
         fuzziness_threshold: fuzziness_threshold for elasticsearch match query 'fuzziness' parameter
-
+        language_script: language of documents to be searched
     Returns:
         dictionary, the search query for ngrams
 
     """
-    term_dict = {
+    term_dict_entity_name = {
         'term': {
             'entity_data': {
                 'value': entity_name
             }
         }
     }
+    term_dict_language = {
+        'term': {
+            'language_script': {
+                'value': language_script
+            }
+        }
+    }
     data = {
         'query': {
             'bool': {
-                'must': [term_dict],
+                'must': [term_dict_entity_name, term_dict_language],
```

Comment:
`create.py` and `populate.py` need changes to add language_script to doc schema and population.
Without that queries will always fail!